prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -236,25 +236,25 @@ open class Receiver(
      */
     fun consistencyErrorMessage(metadata: Metadata): String? {
```

Comment:
This function needs some rework, currently has some dead code, not sure how/why it is necessary. Will investigate briefly, maybe open a tech debt ticket if it doesn't make sense to fix it in this PR.